classes/Course/class.hubCourse.php

```diff
@@ -203,8 +203,8 @@ protected function updateAdditionalFields() {
 
 
 	protected function deleteCourse() {
+        $hist = $this->getHistoryObject();
```

Comment:
this is the only part i need to test a lot since it will change how deleted courses are teated